Servers/UI/OJS.Servers.Ui/ClientApp/src/components/contests/contest-sorting/ContestSorting.module.scss

```diff
@@ -0,0 +1,35 @@
+﻿@import "src/styles/colors";
+@import "src/styles/fonts";
+
+.heading {
+  text-align: left;
+  padding-bottom: 10px;
+  border-bottom: 1px solid $bottom-border-color;
+}
+
+.sortingTypeContainer {
+  margin-top: 10px;
+}
+
+.buttonContainer {
+  display: flex;
+  justify-content: space-between;
+  align-items: end;
+}
+
+.button {
+  font-weight: 500;
```

Comment:
I have overriden it here so that it looks paler, but I removed it now, if you do not think it is necessary. The default font-weight of the button is 700.